wkconnect/backends/neuroglancer/models.py

```diff
@@ -42,6 +42,7 @@ class Layer:
     scales: Tuple[Scale, ...]
     relative_scale: Vec3D
     type: str
+    largestSegmentId: Optional[int] = field(default=None)
```

Comment:
Here `field` is not necessary, since `None` can't be mutated. This is only important for factories, since the default value is class-scoped, and therefore changed for all instances if it is mutable.
```suggestion
    largestSegmentId: Optional[int] =None
```